cmake/modules/OCApplyCommonSettings.cmake

```diff
@@ -11,10 +11,19 @@ function(apply_common_target_settings_soft targetName)
                 )
     endif()
     if(MSVC)
-        # enable linter like warnings with msvc
-        # this includes deprecations
-        # https://learn.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-170
-        target_compile_options(${targetName} PRIVATE /W3)
+        target_compile_options(${targetName} PRIVATE
+                # enable linter like warnings with msvc
```

Comment:
Indentation is wrong.

Reply:
as we still use no linter and have not decided on a code style, there is no such thing as wrong indent https://github.com/owncloud/client/pull/10601